### DISCUSSION TITLE
Does Cosmo open the door to Guix running multi-OS?

### DISCUSSION THREAD

**Discussion by worldofgeese** [jart/cosmopolitan#930] [Q&A] [answered]
I'd take this to the Guix mailing list but I'd like to understand how feasible this is first before I do.

**Comment by jart** [comment, marked as answer]
Last time I tried to build Guix, I seem to recall that it depended on C++ exceptions. That's something Cosmo isn't able to support yet. It'll probably be some time before we're able to support Guix.

**Reply by SerJaimeLannister** [reply]
couldn't you convert that C++ code to llvm  ir and then use llvm-cbe to generate the respective c code?
